www/docs/content/upgrade-guides/medusa-core/1-16-0.md

```diff
@@ -0,0 +1,51 @@
+---
+description: 'Actions Required for v.1.16.0'
+sidebar_custom_props:
+  iconName: 'server-stack-solid'
+---
+
+# v1.16.0
+
+Version 1.16.0 of Medusa introduces breaking changes to database schemas.
+
+## Overview
+
+This release contains a migration that changes the relationship between `MoneyAmount` and `ProductVariant` from Many-to-One to Many-to-Many.
+
+Additionally, the following foreign key constraints have been dropped:
+
+> These changes are **only** relevant if you are using the feature flag `isolate_product_domain`. Please be careful about enabling this flag, as it will introduce experimental changes and work in progress
```

Comment:
```suggestion
:::note

These changes are only relevant if you are using the feature flag `isolate_product_domain`. Please be careful about enabling this flag, as it will introduce experimental and work-in-progress changes.

:::
```